### DISCUSSION TITLE
Update CSME to fix security warning

### DISCUSSION THREAD

**Discussion by slaclau** [fwupd/fwupd#7138] [General]
**Is your feature request related to a problem? Please describe.**
I was previously seeing a warning of HSI:0 until I found a way to update the CSME firmware.

**Describe the solution you'd like**
Update CSME directly from fwupd

**Describe alternatives you've considered**
Manual update using instructions from https://winraid.level1techs.com/t/intel-converged-security-management-engine-drivers-firmware-and-tools-2-15/30719

**Additional context**

**Comment by hughsie** [comment]
We can't upload the firmware I'm afraid, it has to be the OEM.

**Reply by slaclau** [reply]
Fair enough, could this suggestion be added to the CSME check page as an alternate resolution for boards where the OEM no longer provides security fixes?
